src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static void main(String[] args) throws DukeException {
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+         */
+        Scanner sc = new Scanner(System.in);
+        String Hello = "Hello! I'm Duke\nWhat can i do for you?";
+        System.out.println(Hello);
+
+        int flag = 0;
+        ArrayList<task> mylist = new ArrayList<>();
+
+        while(flag == 0) {
+            String input = sc.nextLine();
+            String[] inarr = input.split(" ");
+            String Line = "_________________________________________________________________";
+
+            System.out.println(Line);
+            try {
+                if (input.equals("bye")) {
```

Comment:
Maybe you could change the if else block into a Switch statement, based on the first word in the inputWords array.
The first word (inputWords[0]) could also be stored as a variable if needed, for increased readability.
Example: 
`switch (inputWords[0]) {`
`case "bye":`
`...`
`}`